src/System.Private.ServiceModel/src/System/ServiceModel/Diagnostics/TraceUtility.cs

```diff
@@ -308,6 +308,11 @@ private static bool PropagateUserActivityCore
             }
         }
 
+        static internal string GetCallerInfo(OperationContext context)
+        {
+            return "null";
```

Comment:
This used to fetch the message property RemoteEndpointMessageProperty which doesn't exist in contract. If it wasn't there, then it would return the string "null". This is used with emitting an etw event which we haven't lit up yet. I'll open a bug that GetCallerinfo doesn't return the remote endpoint info. There might be another way to achieve this without this property. If we do implement the property, we should consider adding it to the contract.